packages/ag-charts-website/src/components/site-header/HeaderNav.tsx

```diff
@@ -1,14 +1,14 @@
 import type { MenuItem } from '@ag-grid-types';
 import { SITE_BASE_URL } from '@constants';
+import styles from '@design-system/modules//HeaderNav.module.scss';
```

Comment:
There's a double slash here

```suggestion
import styles from '@design-system/modules/HeaderNav.module.scss';
```